modules/nicknames/events/guildMemberUpdate.js

```diff
@@ -0,0 +1,11 @@
+const {renameMember} = require('../renameMember');
+
+module.exports.run = async function (client, oldGuildMember, newGuildMember) {
+
+    if (!client.botReadyAt) return;
+    if (newGuildMember.guild.id !== client.guild.id) return;
+    if (newGuildMember.nickname === oldGuildMember.nickname && newGuildMember.roles.cache.size === oldGuildMember.guild.me.roles.cache.size) return;
```

Comment:
```suggestion
    if (newGuildMember.nickname === oldGuildMember.nickname && newGuildMember.roles.cache.size === oldGuildMember.roles.cache.size) return;
```
why would we compare the number of roles of the new guild member to the number of roles the client has on the guild?

Reply:
Because our friend autocomplete put things in there I didn't see